coralproject/talk
# [CORL-1077] Rudderstack Integration

## What does this PR do?

Adds support for Rudderstack to forward events outside Coral instances to third party data providers. Added events for:

  - Comment Moderated
  - Comment Created
  - Story Created
  - Comment Reaction Created
  - User Flagged Comment



## What changes to the GraphQL/Database Schema does this PR introduce?

None.



## How do I test this PR?



Configure Rudderstack by providing the following configuration parameters after the [Rudderstack Setup](https://docs.rudderstack.com/administrators-guide/installing-and-setting-up-rudderstack/docker):

- `ANALYTICS_BACKEND_KEY`
- `ANALYTICS_DATA_PLANE_URL`

Perform the events listed in the first section and see the events appear!
